src/app/app/organizations/edit/edit-organization.component.ts

```diff
@@ -27,7 +27,8 @@ export class EditOrganizationComponent implements OnInit {
   // Forms
   editOrgForm: FormGroup = this.formBuilder.group({
     name: [this.organization.name, Validators.required],
-    logoUrl: [this.organization.logoUrl]
+    logoUrl: [this.organization.logoUrl],
+    socialOrganizationName: [this.organization.socialName]
```

Comment:
Can you simplify with `socialName` instead of `socialOrganizationName` please ? As we're editing an organization it is not required to specity it.